install.ps1

```diff
@@ -0,0 +1,84 @@
+# if user isnt admin then quit
+function IsUserAdministrator {
+    $user = [Security.Principal.WindowsIdentity]::GetCurrent()
+    $principal = New-Object Security.Principal.WindowsPrincipal($user)
+    return $principal.IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
+}
+
+if (-not (IsUserAdministrator)) {
+    Write-Host "Please run as administrator"
+    exit 1
+}
+
+
+
+# Replace these values with your tool's information:
+$REPO_OWNER = "ibigio"
```

Comment:
I'm guessing there's a way to dynamically get the user? Would like to avoid people having to edit this file for install.

Reply:
We could do cli args